pkg/systemd/terminal.jsx

```diff
@@ -71,19 +82,31 @@ const _ = cockpit.gettext;
             if (this.state.channel)
                 terminal = (<Terminal ref="terminal"
                      channel={this.state.channel}
+                     theme={this.state.theme}
                      onTitleChanged={this.onTitleChanged} />);
             else
                 terminal = <span>Loading...</span>;
 
             return (
                 <div className="console-ct-container">
-                    <div className="panel-heading">
+                    <div className="panel-heading terminal-group">
                         <tt className="terminal-title">{this.state.title}</tt>
-                        <button ref="resetButton"
-                             className="btn btn-default pull-right"
-                             onClick={this.onResetClick}>{_("Reset")}</button>
+                        <div>
+                            <span>Appearance:</span>
```

Comment:
This should be a `label` with `htmlFor="theme-select"`. Also, the string needs to be i18n'ed.